test/hotspot/jtreg/containers/docker/TestMisc.java

```diff
@@ -57,6 +57,7 @@ public static void main(String[] args) throws Exception {
             testIsContainerized();
             testPrintContainerInfo();
             testPrintContainerInfoActiveProcessorCount();
+            testPrintContainerMemoryInfo("100M", "150M");
```

Comment:
Again. This test runs unconditionally. `--memory-swappiness` is not supported in cgroups v2. Thus, the test will fail on a cgroups v2 system. You need to only run this test on a cgroups v1 system. Have a look at `test/lib/jdk/test/lib/containers/cgroup/MetricsTester.java` how could could detect this and only run on `cgroupv1` providers.

Reply:
On my cgroups v2 system (i.e. it's using 200m memory + 200m swap):
```
$ sudo docker run --rm -ti --memory-swappiness=0 --memory=200m fedora:35
WARNING: Your kernel does not support memory swappiness capabilities or the cgroup is not mounted. Memory swappiness discarded.
[root@fb3182c8e23c /]# cat /sys/fs/cgroup/memory.max 
209715200
[root@fb3182c8e23c /]# cat /sys/fs/cgroup/memory.swap.max 
209715200
```